app/controllers/hiring_staff/vacancies_controller.rb

```diff
@@ -86,4 +83,8 @@ def clear_cache_and_step
   def set_vacancy
     @vacancy = current_school.vacancies.active.find(vacancy_id)
```

Comment:
Now that I have reverted the part of this PR about combining the `job_id` and `id` params together, this will have to wait until we reduce the variety of param names down from 2 to 1.